addons/website_jitsi/models/chat_room.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import uuid
+
+from odoo import api, fields, models
+
+
+class ChatRoom(models.Model):
+    """Store all useful information to manage chat room (Jitsi).
+
+    This model embed all information about the chat room. We do not store them in the
+    mixin to avoid to add too many fields on the models which want to use the chat room
+    mixin (as the behavior can be optional in those models).
+
+    The participant count is automatically updated thanks to the chat room widget.
+    """
+
+    _name = "chat.room"
+    _description = "Chat Room"
+
+    name = fields.Char(
+        "Room Name", required=True, copy=False,
+        default=lambda self: "odoo-room-%s" % str(uuid.uuid4())[:8])
+    is_full = fields.Boolean("Full", compute="_compute_full")
+    lang_id = fields.Many2one(
+        "res.lang", "Language",
+        default=lambda self: self.env["res.lang"].search([("code", "=", self.env.user.lang)], limit=1))
+    max_capacity = fields.Selection(
+        [("4", "4"), ("8", "8"), ("12", "12"), ("16", "16"),
+         ("20", "20"), ("no_limit", "No limit")], string="Max capacity",
+        default="8", required=True)
+    participant_count = fields.Integer("Participant count", default=0, copy=False)
+    # reporting fields
+    last_activity = fields.Datetime("Last activity", readonly=True, default=lambda self: fields.Datetime.now(), copy=False)
+    last_joined = fields.Datetime("Last joined", readonly=True, default=lambda self: fields.Datetime.now(), copy=False)
```

Comment:
this field is only used here : _event_meeting_rooms_get_values
at `meeting_rooms = meeting_rooms.sorted(lambda m: (m.is_pinned, m.room_last_joined, m.id), reverse=True)`
Maybe we could used last_activity instead in this case ? so we could remove last_joined field.
Not sure 'last joined' is a better indicator than 'last activity' to sort the rooms in the list view.

Reply:
Hagrid with you.